Clarification regarding competitors wearing a smartphone on their pockets (#1000)
thewca/wca-regulations: wca-regulations.md

    - 2i2) Competitors may use cameras at the solving station at the discretion of the WCA Delegate, but the following restrictions apply from the start of the attempt until the competitor stops the solve. Penalty for breaking a restriction: disqualification of the attempt (DNF).
        - 2i2a) Each camera monitor must be blank or out of sight of the competitor (see [Regulation A5b](regulations:regulation:A5b)).
        - 2i2b) The competitor must not interact with (e.g. operate, hold, wear) any active camera. Exception: the competitor may wear a camera mounted on their head, as long as it is out of their sight and it is clear that they are not interacting with it (apart from wearing it).

>     - 2i3) The competitor may have a cell phone in their pocket, as long as it is clear that they are not interacting with it (apart from it being on their person).

Author: ```suggestion
    - 2i3) The competitor may have a cell phone in their pocket, as long as it is clear that they are not otherwise interacting with it.
```

>     - 2i3) Exception: the competitor may wear a cell phone on their pocket, as long as it is out of their sight and it is clear that they are not interacting with it (apart from wearing it).

Reviewer: ```suggestion
    - 2i3) Exception: the competitor may have a cell phone in their pocket, as long as it is out of their sight and it is clear that they are not interacting with it (apart from it being on their person).
```

The original doesn't make sense as you typically don't wear a cell phone.

Author: Thank you!

>     - 2i3) Exception: the competitor may have a cell phone in their pocket, as long as it is out of their sight and it is clear that they are not interacting with it (apart from it being on their person).

Author: ```suggestion
    - 2i3) The competitor may have a cell phone in their pocket, as long as it is clear that they are not interacting with it (apart from it being on their person).
```

    - 2i2) Competitors may use cameras at the solving station at the discretion of the WCA Delegate, but the following restrictions apply from the start of the attempt until the competitor stops the solve. Penalty for breaking a restriction: disqualification of the attempt (DNF).
        - 2i2a) Each camera monitor must be blank or out of sight of the competitor (see [Regulation A5b](regulations:regulation:A5b)).
        - 2i2b) The competitor must not interact with (e.g. operate, hold, wear) any active camera. Exception: the competitor may wear a camera mounted on their head, as long as it is out of their sight and it is clear that they are not interacting with it (apart from wearing it).

>     - 2i3) The competitor may have a cell phone in their pocket, as long as it is clear that they are not interacting with it (apart from it being on their person).

Reviewer 2: I'd suggest keeping this simple:
```suggestion
    - 2i3) The competitor may have a cell phone in a pocket, as long as it is clear that they are not interacting with it (apart from having it in a pocket).
```

Could also do "as long as it is clear that they are not otherwise interacting with it."

Reviewer 3: I prefer the alternative you suggested:
"as long as it is clear that they are not otherwise interacting with it."
